libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -56,6 +72,21 @@ export interface IAttributeFilterButtonOwnProps {
      */
     filter?: IAttributeFilter;
 
+    /**
+     * Specifies a parent attribute filter that will be used to reduce options for for current attribute filter.
+     */
+    parentFilters?: ValuesOrPlaceholders<IAttributeFilter>;
+
+    /**
+     * Specifies a parent attribute filter that will be used to reduce options in child attribute filter.
```

Comment:
Specify {@link @gooddata/sdk-ui#IPlaceholder} to use to get and set the value of the attribute filter.
Note: It's not possible to combine this property with "filter" property. Either - provide a value, or a placeholder.

(add similar note about impossible combination to filter property).

Btw - you are not forced to use placeholder only for parent filter. You can use it even for simple attribute filter.